app/models/export/invoices/extract.rb

```diff
@@ -3,8 +3,10 @@ class Invoices
     module Extract
       def self.all_relevant
         SubmissionEntry.invoices
-                       .joins(:submission)
+                       .select('submission_entries.*, frameworks.short_name AS _framework_short_name')
+                       .joins(submission: :framework)
                        .where(submissions: { aasm_state: 'completed' })
```

Comment:
could merge in the scope from the state machine instead to make this slightly safer (i.e. it will blow up if the state is removed/replaced without changing this line): `.merge(Submission.completed)`

Reply:
Oh yes, that'll make the changeover a lot more obvious, thanks